sql/src/main/java/io/crate/metadata/information/InformationKeyColumnUsageTableInfo.java

```diff
@@ -59,27 +63,28 @@
         static final Reference TABLE_SCHEMA = createRef(Columns.TABLE_SCHEMA, DataTypes.STRING);
         static final Reference TABLE_NAME = createRef(Columns.TABLE_NAME, DataTypes.STRING);
         static final Reference COLUMN_NAME = createRef(Columns.COLUMN_NAME, DataTypes.STRING);
-        static final Reference ORDINAL_POSITION = createRef(Columns.ORDINAL_POSITION, DataTypes.SHORT);
+        static final Reference ORDINAL_POSITION = createRef(Columns.ORDINAL_POSITION, DataTypes.INTEGER);
     }
 
-    public static ImmutableMap<ColumnIdent, RowCollectExpressionFactory<Void>> expressions() {
-        return ImmutableMap.<ColumnIdent, RowCollectExpressionFactory<Void>>builder()
+    public static ImmutableMap<ColumnIdent, RowCollectExpressionFactory<InformationSchemaIterables.KeyColumnUsage>> expressions() {
+        //noinspection Convert2MethodRef
```

Comment:
Why aren't you using method refs?

Reply:
Found that they blow up line length here. I find the IntelliJ suggestions a bit too intrusive for this case.

Reply:
In this case I actually agree with Intellij: There is no downside to method refs (well, other than the line length), keeps the code style consistent and it's imho nice for reviews: Method ref syntax can be skipped over more easily. While a lambda might involve closures and other stuff - so I've to read them more carefully

Reply:
ok, i don't mind :)